### ISSUE TITLE
Investigate Validation Loss < Testing Loss

### ISSUE THREAD

**Issue by paul-tqh-nguyen** [paul-tqh-nguyen/react_loading_screens#22]
This is usually a sign of overfitting. Investigate and let's see what's going on.

**Comment by paul-tqh-nguyen** [comment]
We might be overfitting because we're using embedding vectors with size > 100 and many users have only 100 reviews. This is a high priority issue to fix.
